lib/homepage.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:tutorial/constants.dart';
+
+class Homepage extends StatefulWidget {
+  const Homepage({super.key});
+
+  @override
+  State<Homepage> createState() => _HomepageState();
+}
+
+class _HomepageState extends State<Homepage> {
+  int currentPageIndex = 0;
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        // appBar: AppBar(
```

Comment:
remove these comments.  You could add a comment that the pages need to have their own app bar if you feel the need